Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
-  gg <- gg + stat_summary(geom="line", fun.y=mean)
-gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
-gg <- gg + ylab("Continuous PD Marker (IU/L)") + xlab("Dose (mg)")
-gg + facet_grid(~DAY_label)
-```
-
-## R Session Info
-```{r}
-sessionInfo()
+---
+title: "PD, Dose-Response - Continuous Endpoint"
+author: "Camille Vong"
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    code_folding: hide
+---
+
+## Overview
+
+This document contains exploratory plots for multiple ascending continuous PD data as well as the R code that generates these graphs. The plots presented here are based on simulated data ([see: PKPD Datasets](PKPD_Datasets.html)). Data specifications can be accessed on [Datasets](Datasets.html) and Rmarkdown template to generate this page can be found on [Rmarkdown-Template](Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd). You may also download the Multiple Ascending Dose PK/PD dataset for your reference ([download dataset](Data/Multiple_Ascending_Dose_Dataset2.csv)).
+
+## Setup
+
+```{r, echo = TRUE, warning=FALSE, message=FALSE}
+library(ggplot2)
+library(dplyr)
+library(tidyr)
+library(xgxr)
+
+#flag for labeling figures as draft
+status = "DRAFT"
+
+## ggplot settings
+xgx_theme_set()
+```
+
+## Load Dataset
+
+```{r, warning=FALSE, message=FALSE}
+#load dataset
+pkpd_data <- read.csv("../Data/Multiple_Ascending_Dose_Dataset2.csv")
+
+#ensure dataset has all the necessary columns
+ pkpd_data = pkpd_data %>%
+    mutate(ID      = ID,     #ID   column
+           TIME    = TIME,   #TIME column name 
+           NOMTIME = NOMTIME,#NOMINAL TIME column name
+           EVID    = EVID   ,#EVENT ID, >=1 is dose, otherwise measurement
+           PROFDAY = PROFDAY,#PROFILE DAY day associated with profile, e.g. day of dose administration
+           LIDV    = LIDV,   #DEPENDENT VARIABLE column name
+           CENS    = CENS,   #CENSORING column name
+           CMT     = CMT,    #COMPARTMENT column
+           DOSE    = DOSE,   #DOSE column here (numeric value)
+           TRTACT  = TRTACT, #DOSE REGIMEN column here (character, with units),
+           LIDV_NORM = LIDV/DOSE,
+           LIDV_UNIT    = ifelse(CMT==2, "ng/ml",
+                                 ifelse(CMT==3, "IU/L", NA )
+                                 )
+    )
+#create a factor for the treatment variable for plotting
+pkpd_data = pkpd_data %>%
+  arrange(DOSE) %>%
+  mutate(TRTACT_low2high = factor(TRTACT, levels = unique(TRTACT)),
+         TRTACT_high2low = factor(TRTACT, levels = rev(unique(TRTACT))))
+
+#create pd dataset
+pd_data <- pkpd_data %>%
+  filter(CMT==3)
+
+
+#units and labels
+  time_units_dataset = "hours"
+  time_units_plot    = "days"
+  trtact_label       = "Dose"
+  dose_label         = "Dose (mg)"
+  conc_label         = "Concentration (ng/ml)"  
+  pd_label           = "Continuous PD Marker (IU/L)"  
+  concnorm_label     = "Normalized Concentration (ng/ml)/mg"

```

Comment:
conc_label, concnorm_labl, conc_units, AUC_units are not needed for a page that only has PD

Reply:
removed now